operatorcourier/api.py

```diff
@@ -26,13 +24,15 @@ def build_and_verify(source_dir=None, yamls=None, ui_validate_io=False,
     """Build and verify constructs an operator bundle from
     a set of files and then verifies it for usefulness and accuracy.
 
-    It returns the bundle as a string.
+    It returns the bundle as a string, or None if the source_dir is nested.
 
     :param source_dir: Path to local directory of yaml files to be read.
     :param yamls: List of yaml strings to create bundle with
     :param ui_validate_io: Optional flag to test operatorhub.io specific validation
     :param validation_output: Path to optional output file for validation logs
     :param repository: Repository name for the application
+    :param output: The path to the operator bundle yaml output. This option is effective
+                   only if the source_dir is not in the nested structure
```

Comment:
Why? What produces build for nested structure?

Reply:
Nothing. This IMO needs to be refactored more significantly because returning a bundle doesn't really make sense in this case anymore.